docs/source/customisation.rst

```diff
@@ -128,119 +128,119 @@ and point your browser to `http://localhost:6543` to see the result.
+  The pattern looks like a http URI with the marker `%s` as a placeholder for 
+  your concept or collection id. Continuing our example, this could be 
+  `http://id.me.org/colours/%s` or `http://data.me.org/colours/%s#id`.
+* `default language`: The default language of the provider. When a concept is 
+  requested, a single label is always returned for ease of display. Normally 
+  this is determined from browser settings or cookies, but if none of these are
+  present, Atramhasis will try this language. Should generally be set to the 
+  main or prevalent language in your thesaurus. When not present, Atramhasis
+  falls back to English as the lingua franca.
+* `force diplay language`: Even though the client can generally choose which 
+  language a concept or tree view should be displayed in, there are times when
+  it's handy to force the issue. This has been used before for biological 
+  taxonomies where you might always want to display the name in Latin,
+  even though there's a more local name available. See `force_display_label_language`_ 
+  for more information. For most thesauri this will be blank.
+* `ID Generation Strategy`: Every concept or collection needs a unique 
+  identifier. This identifier is used in generating URI's. There are three
+  possible strategies for generating ID's:
+
+  * `numeric (default)`: Before Atramhasis 2.0.0 this was the only option. 
+    It generates a numeric id based on the current highest id in the conceptscheme.
+  * `guid`: Generate a GUID as identifier for your concept or collection.
+  * `manual`: No identifier is generated by Atramhasis, but the user
+    is required to generate them manually. Allows for full control
+    over ID's and URI's, at the cost of a little more work.
+
+* `subject`: Add a certain tag to a provider. Currently there are two
+  recognised tags:
+  
+  * `external`: Must be added to an external provider, in code.
+  * `hidden`: Used to obfuscate a conceptscheme. The scheme will not be
+    shown in the Atramhasis UI, except when the URL's of concepts or
+    collections are requested. This allows having a conceptscheme that
+    can be used in applications, but is not the main focus of your 
+    Atramhasis instance. See `hidden_providers`_ for more information.
+
+* `expand strategy`: One of the features of Atramhasis is quickly 
+  determining all the narrower concepts of a certain concept, mainly to 
+  be used in querying other systems. There are two options:
+
+  * `recurse`: Use recursive queries to determine the narrower concepts.
+    For large vocabularies this generally requires good caching.
+  * `visit`: Query a special visitation table that is pre-computed. Will
+    be much more performant, but requires that the table is recomputed
+    regulary.
+  
+  When in doubt, use recurse and pay attention to caching.
+
+
+After filling out the required fields, and pressing `Save` a provider and
```

Comment:
`required fields, and pressing Save a provider` -->  `required fields and pressing Save, a provider`
(move comma)